botocore/data/autoscaling/2011-01-01/waiters-2.json

```diff
@@ -0,0 +1,62 @@
+{
+    "version": 2,
+    "waiters": {
+        "GroupExists": {
+            "acceptors": [
+                {
+                    "argument": "length(AutoScalingGroups) > `0`",
```

Comment:
We should probably discuss this.  This is misleading if you specify multiple groups.  Autoscaling will just omit that group from its returned list.